simulation/modules/afumigatus.py

```diff
@@ -83,3 +86,13 @@ def is_branchable(self, branch_probability):
             super().is_branchable(branch_probability) &
             (self.cells['status'] == Status.HYPHAE)
         )
+
+
+@attr.s(kw_only=True)
+class AfumigatusState(ModuleState):
+    tree = cell_tree(AfumigatusCellTree)
+
```

Comment:
I am kind of confused over here. If I understand it correctly, the AfumigatusState only has one tree and one tree only has one root. However, there might be multiple aspergillus spores growing into hyphae across the simulation, so I believe it will end up with a set of trees.